src/locales/en.json

```
@@ -683,6 +683,18 @@
     "generic": "My invoices"
   },
 
+  "PersonalInfo": {
+    "birthcity": "Birth city",
+    "nationality": "Nationality",
+    "save-cta": "Save information",
+    "modal-title": "Edit personal information",
+    "info-saved-succesfully": "Your personal info has been updated",
+    "info": {
+      "title": "Why is this information necessary ?",
```

```suggestion
      "title": "Why is this information necessary?",
```

Spaces before question marks is a french thing :)